modules/servodyn/src/ServoDyn_Registry.txt

```diff
@@ -133,6 +133,7 @@ typedef	^	BladedDLLType	IntKi	GenState	-	-	-	"Generator state from Bladed DLL"	N
 #typedef	^	BladedDLLType	ReKi	ElecPwr	-	-	-	"Electrical power sent to Bladed DLL"	W
 typedef	^	BladedDLLType	ReKi	BlPitchCom	3	-	-	"Commanded blade pitch angles"	radians
 typedef	^	BladedDLLType	ReKi	PrevBlPitch	3	-	-	"Previously commanded blade pitch angles"	radians
+typedef	^	BladedDLLType	ReKi	BlFlapCom	3	-	-	"Commanded lap deflection angles"	degrees
```

Comment:
I **highly recommend** using radians instead of degrees. Everything else in the Bladed interface and everything that is passed between modules in the FAST code uses radians.

Reply:
I set this to radians and changed the names to more generalized names for those channels.